### ISSUE TITLE
systemd-firstboot: Tell the service manager about the system locale

### ISSUE THREAD

**Issue by AdrianVovk** [systemd/systemd#17216]
**Is your feature request related to a problem? Please describe.**
I noticed that when I first install my OS, gnome-terminal refuses to start. It turns out, gnome-terminal-server.service does not know what locale the system is set to and exits. Looking at the code of systemd-firstboot, there is nothing I can see that adds LANG= into the service environment.

**Describe the solution you'd like**
Either have systemd-firstboot do the equiv. of `systemctl set-environment LANG=[...]` or have systemd-firstboot use localed directly

**Describe alternatives you've considered**
For now, I have this workaround:
```
ExecStartPost=/usr/bin/bash -c "systemctl set-environment $(cat /etc/locale.conf)"
```

**Comment by poettering** [comment]
true, we need to trigger the vconsoled/localed/hostnamed logic when running on a live system at boot. It's a bug that we do not.

**Comment by poettering** [comment]
Fixed by d0c50d8dfa7ea6721d3b83af5bbbd4d96be22085, closing
